packages/domi/test/MDVector/MDVector_UnitTests.cpp

```diff
@@ -871,21 +873,15 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( MDVector, augmentedConstruction, Sca )
 #endif
 
 #ifdef HAVE_TPETRA
-  Teuchos::RCP< Tpetra::Vector< Sca, int >      > tv1  =
-    mdv1.template getTpetraVectorView< int >();
-  Teuchos::RCP< Tpetra::MultiVector< Sca, int > > tmv2 =
-    mdv2.template getTpetraMultiVectorView< int >();
-  Teuchos::RCP< Tpetra::MultiVector< Sca, int > > tmv3 =
-    mdv3.template getTpetraMultiVectorView< int >();
+  Teuchos::RCP< Tpetra::Vector< Sca, int >      > tv1  = mdv1.template getTpetraVectorView< int >();
```

Comment:
`int` is the default (and only permitted) value of `LocalOrdinal`, so you don't need to specify `int` as the second template argument of `Tpetra::Vector` and `Tpetra::MultiVector`.

Reply:
I have changed these tests to use the simplified template arguments.